tests/test_utils.h

```diff
@@ -25,10 +27,11 @@ static inline void vhd_log_stderr(enum LogLevel level, const char *fmt, ...)
     if (level <= LOG_VERBOSITY) {
         char timestr[64];
         struct timeval tv;
+        struct tm local_tm;
 
         gettimeofday(&tv, NULL);
-        strftime(timestr, sizeof(timestr), "%F %T", localtime(&tv.tv_sec));
-        fprintf(stderr, "%s.%03ld [%8s] ", timestr, tv.tv_usec / 1000,
+        strftime(timestr, sizeof(timestr), "%F %T", localtime_r(&tv.tv_sec, &local_tm));
+        fprintf(stderr, "%s.%06ld [%8s] ", timestr, tv.tv_usec / 1000,
```

Comment:
microseconds have six digits

Reply:
Do we need this precision in our logs?

Reply:
confused with timespec, there were no problems here